src/main/java/mate/academy/model/User.java

```diff
@@ -53,8 +57,25 @@
     @Override
     public String toString() {
         return "User{"
-            + "id=" + id
-            + ", email='" + email + '\''
-            + '}';
+                + "id=" + id
+                + ", email='"
+                + email + '\''
+                + '}';
```

Comment:
Consider excluding `password` and `salt` from the `toString` method to avoid logging or displaying sensitive information.